### PR TITLE
Revert ransomware feature config changes

### PR DESCRIPTION
**Pull request by bsuryadevara**
Reverted ransomware feature config changes as dask is overriding config values.

### CODE REVIEW

File: examples/ransomware_detection/stages/create_features.py [hunk 1]
```diff
@@ -59,10 +59,9 @@ def __init__(
         threads_per_worker: int = 2,
     ):
         self._client = Client(threads_per_worker=threads_per_worker, n_workers=n_workers)
+        self._feature_config = FeatureConfig(file_extns, interested_plugins)
+        self._feas_all_zeros = dict.fromkeys(feature_columns, 0)
```
Comment (mdemoret-nv) [inline examples/ransomware_detection/stages/create_features.py:63]: Is `self._feas_all_zeros` used anywhere else? Do we need to save it in `self`?

**Review by mdemoret-nv — APPROVED** [review]
Just one question. I like this better than saving `_config`

File: examples/ransomware_detection/stages/create_features.py (hunk 1 above)
Comment (bsuryadevara) [reply examples/ransomware_detection/stages/create_features.py:63]: using it here: https://github.com/nv-morpheus/Morpheus/blob/7b0bb90431921041954d4b5ebe75d23583110eeb/examples/ransomware_detection/stages/create_features.py#L114

**Comment by mdemoret-nv** [comment]
@gpucibot merge

**Pull request merged**